### DISCUSSION TITLE
Fine-tuning bert for NER

### DISCUSSION THREAD

**Discussion by rajatjaiswal96** [NVIDIA-NeMo/NeMo#4290] [Q&A]
I wanted to do fine-tuning of BERT model for NER task. 
I tried to look at the example NER script for finetuning but following that it's not working as model configs are not getting updated while fitting the trainer with my own dataset labels. i can still see old labels on which model is pretrained on.
What will be the corresponding steps to finetune NER model on my won dataset, is there any documentation or example script for that ?

**Comment by rajatjaiswal96** [comment]
@titu1994 @piraka9011 any suggestion ?

**Reply by ericharper** [reply]
Have you seen this tutorial? https://github.com/NVIDIA/NeMo/blob/r1.10.0/tutorials/nlp/Non_English_Downstream_Tasks_(NER).ipynb

Please let us know if it helps.
